Fix incorrect man page and include mention of the possibility
of linking to a static libary only.
https://bugs.gentoo.org/show_bug.cgi?id=562570

--- a/primegen.3
+++ b/primegen.3
@@ -14,13 +14,16 @@
 .br
 void \fBprimegen_skipto\fP(&\fIpg\fR,\fIbound\fR);
 
-struct primegen \fIpg\fR;
+primegen \fIpg\fR;
 .br
 uint64 \fIbound\fR;
 .SH DESCRIPTION
 The primegen library generates prime numbers in order:
 2, 3, 5, 7, etc.
 It can generate primes as large as 10^15.
+Only a static version of this library is provided, so instead of
+linking your program with -lprimegen you will have to pass the
+full path to the library (e.g. /usr/lib/primegen.a) to the compiler.
 
 The simplest way to use primegen
 is to call
